examples/amm/src/contract.rs

```diff
@@ -346,6 +352,7 @@ impl Amm {
 
     fn execute_application_call_remote(
         &mut self,
+        runtime: &mut ContractRuntime<Abi>,
         outcome: &mut ExecutionOutcome<Message>,
```

Comment:
We kinda have the worst of both worlds right after this PR :-|

Reply:
Updated. Hopefully it's better now :crossed_fingers: